samples/join-a-team-using-qr-code/nodejs/index.js

```diff
@@ -0,0 +1,108 @@
+const path = require('path');
+const express = require('express');
+const cors = require('cors');
+const ENV_FILE = path.join(__dirname, '.env');
+require('dotenv').config({ path: ENV_FILE });
+const PORT = process.env.PORT || 3978;
+const server = express();
+
+server.use(cors());
+server.use(express.json());
+server.use(express.urlencoded({
+    extended: true
+}));
+server.engine('html', require('ejs').renderFile);
+server.set('view engine', 'ejs');
+server.set('views', __dirname);
+
+// Import required bot services.
+// See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const { BotFrameworkAdapter, ConversationState, MemoryStorage, UserState } = require('botbuilder');
+
+const { TeamsBot } = require('./bots/teamsBot');
+const { MainDialog } = require('./dialogs/mainDialog');
+const { teamData } = require('./bots/dialogBot');
+const QRCode = require('qrcode')
+
+// Create adapter.
+// See https://aka.ms/about-bot-adapter to learn more about adapters.
+const adapter = new BotFrameworkAdapter({
+    appId: process.env.MicrosoftAppId,
+    appPassword: process.env.MicrosoftAppPassword
+});
+
+adapter.onTurnError = async (context, error) => {
+    // This check writes out errors to console log .vs. app insights.
+    // NOTE: In production environment, you should consider logging this to Azure
+    //       application insights. See https://aka.ms/bottelemetry for telemetry 
```

Comment:
Do we need to put this commented code?

Reply:
It's added in previous samples also which give details about how the general bot errors can be captured and what can be used for deployed environment.